build.gradle

```diff
@@ -42,11 +42,11 @@ subprojects {
             instructionReplace "Bundle-Vendor", "Reactive Streams SIG"
             instructionReplace "Bundle-Description", "Reactive Streams API"
             instructionReplace "Bundle-DocURL", "http://reactive-streams.org"
-            instructionReplace "Bundle-Version", "1.0.1-RC2"
+            instructionReplace "Bundle-Version", "1.0.1"
         }
     }
 
-    if (name in ["reactive-streams", "reactive-streams-tck", "reactive-streams-examples"]) {
+    if (name in ["reactive-streams", "reactive-streams-tck", "reactive-streams-examples", "reactive-streams-flow-bridge"]) {
```

Comment:
Not to start a bikeshedding exercise, but are we on board with the naming of this artifact?

Options are things like "reactive-streams-flow-compat", "reactive-streams-jdk9-compat", "reactive-streams-jdk9-interop" and the like.

Reply:
Heh yeah did not want to open that bikeshed as I've bikeshedded quite a lot here already.
Only precedent for such a library would be the https://github.com/scala/scala-java8-compat library I think, and the inclusion of the number there perhaps is sub optimal. So out of the ones you proposed I like `reactive-streams-flow-compat` I guess, or `reactive-streams-java-flow-compat`.

It's not super important to me though I guess, so would be interesting to hear or do a quick vote I guess?

Reply:
How about reactive-streams-jsr266-interop?